apps/portals/arcusbio/src/types.d.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference types="vite/client" />
+interface ImportMetaEnv {
+  readonly VITE_PORTAL_NAME: string
+  readonly VITE_PORTAL_DESCRIPTION: string
+  readonly VITE_PORTAL_KEY: string
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv
+}
+
+// Import MUI type augmentations from 'synapse-react-client' so we can use custom property values defined for our MUI theme
+import 'synapse-react-client/ThemeTypes'
+
+declare module '*.module.scss' {
+  const classes: { readonly [key: string]: string }
+  export default classes
+}
```

Comment:
Is this necessary? I don't think we had to add these types to any other project where we've already started using SCSS modules.